src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,330 @@
+    private static final String DEADLINE_FORMAT = "Format: deadline {your task} /by {deadline date}\n";
+    private static final String EVENT_USAGE = "event: to add an event task in your list";
+    private static final String EVENT_FORMAT = "Format: event {your task} /from {begin date} /to {end date}\n";
+    private static final String DELETE_USAGE = "delete: to delete a task";
+    private static final String DELETE_FORMAT = "Format: delete {Number}\n";
+    private static final String ADDING_TASK = "Got it. I've added this task:";
+    private static final String BIG_NUMBER = "The task number is bigger than the number of tasks";
+    private static final String FINISH_UNMARKING_MESSAGE = "Ok! I've marked this task as not done yet:";
+    private static final String FINISH_MARKING_MESSAGE = "Nice! I've marked this task as done:";
+    private static final String ASKING_MESSAGE = "What can I do for you?";
+    public static final String INCORRECT_KEYWORD = "OOPS!!! I'm sorry, but I don't know what that means :-(";
+    public static final String EMPTY_TODO_DESCRIPTION = "OOPS!!! The description of a todo cannot be empty.";
+    public static final String EMPTY_DEADLINE_DESCRIPTION = "OOPS!!! The description of a deadline cannot be empty.";
+    public static final String EMPTY_EVENT_DESCRIPTION = "OOPS!!! The description of a event cannot be empty.";
+    public static final String INVALID_INPUT = "This is an invalid input, please follow this input format\n";
+    public static final String DEADLINE_INVALID_INPUT = INVALID_INPUT + DEADLINE_FORMAT;
+    public static final String EVENT_INVALID_INPUT = INVALID_INPUT + EVENT_FORMAT;
+    public static final String MEANINGLESS_SENTENCE_AFTER_KEYWORD = "OPPS!!! The sentence after keyword has no meaning";
+    public static final String REMOVE_MESSAGE = "Noted. I've removed this task:";
+    public static final String WRITEFILE_EXCEPTION_MESSAGE = "OPPS!!! Something went wrong when you write to data file";
+    public static final ArrayList<String> KEYWORDS = new ArrayList<>(
+            List.of("todo","deadline","event","list","bye","mark","unmark","help","delete")
+    );
+
+    public static void main(String[] args) {
+        ArrayList<Task> tasks = new ArrayList<>();
+        FileClass dataFile = new FileClass("src/main/data.txt", tasks);
+        try {
+            dataFile.addTasks();
+        } catch (FileNotFoundException e) {
+            System.out.println("Please give a correct data file path");
+            return;
+        }
+        showWelcomeMessage();
+
+        String userInput;
+        Scanner in = new Scanner(System.in);
+
+        while (true) {
+            userInput = getInput(in);
+            String[] separatedKeyWordAndContent = userInput.split(" ", 2);
+            String keyword = separatedKeyWordAndContent[0];
+            if (!KEYWORDS.contains(keyword)) {
+                System.out.println(INCORRECT_KEYWORD);
```

Comment:
It's good that you are using named constant here. You might also want to named them to sound more intuitive (e.g. `INCORRECT_KEYWORD_ERROR_MESSAGE`). Same goes to other similar variable names.